api/src/pcapi/static/backofficev3/js/addons/pc-confirm-modal.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Store an input initial state and restore if modal gets dismissed.
+ *
+ * We have two behaviors:
+ *
+ * 1. checkbox: will store the `checked` as initial value
+ * 1. other inputs: will store the `value` as initial value
+ *
+ * Add `data-confirm-modal=".my-modal-123"` on your input with `.my-modal-123` being a unique class on your `.modal`.
```

Comment:
```suggestion
 * Add `data-confirm-modal=".my-modal-123"` in your input with `.my-modal-123` being a unique class on your `.modal`.
```

Reply:
T'es sur ? C'est sur l'input pas dedans (`on` plutôt que `in`.)